### PR TITLE
Sanitize html

### PR DESCRIPTION
**Pull request by shrutiyer**
Addressed #101 issue
NOTE: Should merge the delUser PR before this

### CODE REVIEW

**Comment by aconsilvio** [comment]
👍

File: public/views/content/settings.html [hunk 1]
```diff
@@ -27,7 +31,7 @@
 
         <span ng-show="passwordMatchError" class="help-block">{{passwordMatchError}}</span>
 
-        <div class="alert alert-info" ng-show="passwordChangeMsg">
+        <div class="alert alert-info form-group" ng-show="passwordChangeMsg">
```
Comment (davidabrahams) [inline public/views/content/settings.html:34]: does this have to be a formgroup?

**Comment by davidabrahams** [comment]
One comment, otherwise :+1:

**Comment by BrennaManning** [comment]
:+1:

File: public/views/content/settings.html (hunk 1 above)
Comment (shrutiyer) [inline public/views/content/settings.html:34]: For the spacing between the alert and the submit button

**Pull request merged**